layersvt/screenshot.cpp

```diff
@@ -683,6 +683,25 @@ static bool writePPM(const char *filename, VkImage image1) {
             destformat = VK_FORMAT_R8G8B8_UNORM;
     }
 
+    // From vulkan spec:
+    //   VUID-vkCmdBlitImage-srcImage-00229
+    //     If either of srcImage or dstImage was created with a signed integer VkFormat,
+    //     the other must also have been created with a signed integer VkFormat
+    //   VUID-vkCmdBlitImage-srcImage-00230
+    //     If either of srcImage or dstImage was created with an unsigned integer VkFormat,
+    //     the other must also have been created with an unsigned integer VkFormat
+    // If the destination format is not compatible, set destintation format to source format and print a warning.
+    if (FormatIsSINT(format) || FormatIsSINT(destformat) || FormatIsUINT(format) || FormatIsUINT(destformat)) {
```

Comment:
How does this make sure that the src/dst signed-ness is the same? What I see is that if the format is signed or unsigned or if the destFormat is signed or unsigned, make the destFormat the same as the format.

Reply:
Went through all the possible case with Charles, and yes, setting destFormat to format is correct and insures consistent signed-ness. But his comments made me realize there is one problem here: The warning message is printed even when the destFormat is not changed. I'll need to fix that.